modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -0,0 +1,169 @@
+                        "35.155.66.53/32",
+                        "44.231.249.237/32",
+                        "44.233.161.100/32",
+                        "3.229.185.234/32",
+                        "54.147.98.63/32",
+                        "54.163.100.197/32",
+                        "23.20.194.86/32",
+                        "23.22.242.238/32",
+                        "54.147.218.140/32",
+                        "52.21.229.141/32",
+                        "54.227.5.10/32",
+                        "54.146.138.135/32",
+                        "23.21.239.1/32",
+                        "52.20.145.130/32",
+                        "54.157.89.24/32",
+                        "107.22.162.110/32",
+                        "3.223.147.2/32",
+                        "54.88.225.66/32",
+                        "54.177.143.128/32",
+                        "54.219.250.189/32",
+                        "18.135.14.84/32",
+                        "18.135.218.119/32",
+                        "3.11.51.55/32",
+                        "3.233.86.181/32",
+                        "34.226.229.129/32",
+                        "44.194.44.138/32",
+                        "34.216.5.118/32",
+                        "52.11.52.9/32",
+                        "52.40.203.77/32",
+                        "18.217.173.123/32",
+                        "3.13.34.119/32",
+                        "3.19.160.232/32",
+                        "18.188.187.143/32",
+                        "18.216.245.132/32",
+                        "52.14.85.89/32",
+                        "52.21.184.148/32",
+                        "44.194.57.131/32",
+                        "18.188.65.148/32",
+                        "3.130.101.176/32",
+                        "3.130.198.233/32",
+                        "54.210.128.71/32",
+                        "54.227.100.14/32",
+                        "54.92.188.93/32",
+                        "107.22.5.187/32",
+                        "3.217.212.27/32",
+                        "52.22.56.3/32",
+                        "52.5.10.152/32",
+                        "54.237.41.201/32",
+                        "34.202.145.72/32",
+                        "52.205.239.95/32",
+                        "54.236.208.68/32",
+                        "3.234.64.191/32",
+                        "34.195.159.252/32",
+                        "34.228.34.122/32",
+                        "54.205.89.242/32",
+                        "209.132.0.0/16",
+                        "66.187.0.0/16",
+                        "2620:0052:0004:0000:0000:0000:0000:0000/48"    
+                    ]
+                },
+                "Bool": {
+                    "aws:ViaAWSService": "false"
+                }
+            }
+        }
+    ]
+}
+----
++
+[NOTE]
+====
+This list is subject to change. Additionally, you must specify the IP addresses in CIDR notation.  
+====
++
+. In the Create policy page, click *Review and create*.
```

Comment:
```suggestion
. Click *Review and create*.
```
(Same note about "In" to "On" if you keep the initial text that restates what page the user is on.